__tests__/app.test.js

```diff
@@ -29,4 +29,24 @@ describe("GET requests", () => {
         });
       });
   });
+  test("200: returns api documentaion", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        // I think this should work but can't work out why it doesnt, please point me to the right notes to fix it.  
+        //for (const key in response.body) {
+        //   expect(key).toHaveProperty(['description']);
+        // }
+        expect(response.body["GET /api"].description).toEqual(
```

Comment:
These tests are fine, but you can just require in the JSON file into the test, and check that is equal to the response!  That's the way to make the test a bit more efficient.